bdsf/gausfit.py

```diff
@@ -398,15 +398,15 @@ def fit_island(self, isl, opts, img, ngmax=None, ffimg=None, ini_gausfit=None):
             # If all else fails, try to use moment analysis
             if verbose: print('All else has failed')
             inisl = N.where(~isl.mask_active)
-            mask_id = N.zeros(isl.image.shape, dtype=N.int32) - 1
+            mask_id = N.zeros(isl.image.shape, dtype=int) - 1
```

Comment:
Here you double the size of `mask_id` , because `dtype=int` is an alias for `dtype=int64`.

Reply:
Yeah, I realized that, but it shouldn't matter in practice (the mask is not a large array).